apiserver/plane/api/urls/issue.py

```
@@ -7,7 +7,8 @@
     IssueCommentAPIEndpoint,
     IssueActivityAPIEndpoint,
     WorkspaceIssueAPIEndpoint,
-    IssueAttachmentEndpoint,
+    IssueAttachmentV2Endpoint,
+    IssueTypeAPIEndpoint
```

Ruff (0.8.2)

`plane.api.views.IssueTypeAPIEndpoint` imported but unused

Remove unused import: `plane.api.views.IssueTypeAPIEndpoint`

(F401)